devtools/maven/src/main/java/io/quarkus/maven/QuarkusBootstrapMojo.java

```diff
@@ -135,6 +135,12 @@
     @Parameter(defaultValue = "${mojoExecution}", readonly = true, required = true)
     private MojoExecution mojoExecution;
 
+    /**
+     * Whether to close the bootstrapped applications after the execution
+     */
+    @Parameter(property = "closeBootstrappedApp")
```

Comment:
I think we should start using `quarkus.` prefix for all mojo parameters because otherwise there can be clashes with other plugins if using `-D` or project-wide `properties`.
If we do so, we should make sure there is no `Unrecognized configuration key` warning. #24585 might have fixed that already, haven't checked yet.

Btw, you could also set `name` explicitly via a constant and use that constant in `DevMojo`, making it more robust in case of changes. Just saying.

Reply:
That was actually the reason I didn't use the `quarkus.` prefix, since it's not related to the app config but may look like one.

Reply:
I renamed it to `quarkusCloseBootstrappedApp` and used a constant where it's used. Thanks @famod

Reply:
Have you checked whether `quarkus.` would be a problem? I fell off my radar, sorry, but I can check tomorrow.

Btw, I think adding `quarkus` to each property member name adds a lot of noise. I'd only use it for the `property` attribute.

Reply:
I renamed the param to `closeBootstrappedApp` and kept the property `quarkusCloseBootstrappedApp`. Using `quarkus.` I am getting a warning about an unrecognized config key. That's only if I set it on the command line though, I am not using the property in the code and it's kind of an "internal" parameter anyway.